core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaSchemaRegistry.java

```diff
@@ -218,9 +179,9 @@ public void init() throws SchemaRegistryInitializationException {
     }
 
     try {
-      createZkNamespace();
-      masterElector = new ZookeeperMasterElector(zkUtils, myIdentity, this,
+      masterElector = new ZookeeperMasterElector(config, myIdentity, this,
                                                  isEligibleForMasterElector);
```

Comment:
My follow up that will add a new elector just does this in `init()`. Not sure if it is worth adding another method here, though tbh I'm not at all happy with how the various initialization happens either (across a bunch of classes)... I think there is a lot of cleanup/reorganization that could help here, especially in the directly coupled relationship between the MasterElectors and KafkaSchemaRegistry that both make calls into each other.